peerprep/app/questions/new/page.tsx

```diff
@@ -132,32 +132,32 @@ function NewQuestion({}: Props) {
           disabled={loading}
           label="Description: "
           name="description"
-          value={formData.description}
+          value={formData.content}
```

Comment:
Referencing `handleFormTextInput` at Line 66, the name of this `FormTextInput` needs to be renamed to match the struct's field - so name needs to be renamed `content`.

Reply:
?